control-plane-operator/hostedclusterconfigoperator/controllers/resources/monitoring/config.go

```diff
@@ -1,45 +0,0 @@
-package monitoring
-
-import (
-	"fmt"
-
-	cmoconfig "github.com/openshift/hypershift/thirdparty/clustermonitoringoperator/pkg/manifests"
```

Comment:
Can we delete this package now that it is unused?

Reply:
we still have another file in there
https://github.com/openshift/hypershift/blob/main/control-plane-operator/hostedclusterconfigoperator/controllers/resources/monitoring/servicemonitor.go

Reply:
I meant the github.com/openshift/hypershift/thirdparty/clustermonitoringoperator/pkg/manifests package

Reply:
(which btw has another bug :) )
I just realized we're only keeping etcd_object_counts ... it really should look like the standalone kube-apiserver service monitor

Reply:
ah, yes, thx